docs/Deploying/Configuration.md

```diff
@@ -2462,6 +2462,87 @@ The content of the PEM-encoded private key for the Simple Certificate Enrollment
       -----END RSA PRIVATE KEY-----
   ```
 
+##### apple_bm_server_token
+
+This is the path to the Apple Business Manager encrypted server token (a `.p7m` file). This is typically generated via `fleetctl generate mdm-apple-bm`. Only one of `apple_bm_server_token` and `apple_bm_server_token_bytes` can be set.
```

Comment:
This is downloaded from apple business manager. `fleetctl generate mdm-apple-bm` only generates the certificate and key pair.

Reply:
Ah gotcha, thanks!